app/views/hq/sponsors/_form.html.haml

```diff
@@ -0,0 +1,105 @@
+= form_for [:hq, @sponsor] do |f|
+  = render 'shared/errors', object: f.object
+
+  .form-group
+    = f.label :name
+    = f.text_field :name, class: "form-control"
+  .form-group
+    = f.label :status_id
+    = f.collection_select :status_id, @statuses, :id, :name, {}, 
+          class: "form-control"
+  .form-group
+    = f.label :gender
+    = f.select :gender, Settings.lookup.gender, {}, class: "form-control"
+  .form-group
+    = f.label :start_date
+    = f.text_field :start_date, class: "form-control date-picker"
+  .form-group
+    = f.label :requested_orphan_count
+    = f.text_field :requested_orphan_count, class: "form-control"
+  - unless f.object.new_record?
+    .form-group
+      = f.label :request_fulfilled
+      = f.check_box :request_fulfilled, {disabled: true, class: "checkbox"},
+           f.object.request_fulfilled ? 1 : 0
+
+
+  - if f.object.new_record?    
```

Comment:
this is where a facade would help - you could have a method #class_for_attribute rather than a conditional for the view.
Without a facade i'm tempted to put this into a view helper